pkg/api/validation/validation.go

```diff
@@ -1231,6 +1231,30 @@ func validateScaleIOVolumeSource(sio *api.ScaleIOVolumeSource, fldPath *field.Pa
 	if sio.VolumeName == "" {
 		allErrs = append(allErrs, field.Required(fldPath.Child("volumeName"), ""))
 	}
+	if sio.SecretRef != nil {
+		if sio.SecretRef.Name == "" {
```

Comment:
Same here, this is tightening validation of existing objects, which isn’t allowed